src/backend/distributed/shardsplit/shardsplit_decoder.c

```diff
@@ -47,9 +64,10 @@ void
 _PG_output_plugin_init(OutputPluginCallbacks *cb)
 {
 	LogicalOutputPluginInit plugin_init =
-		(LogicalOutputPluginInit) (void *) load_external_function("pgoutput",
-																  "_PG_output_plugin_init",
-																  false, NULL);
+		(LogicalOutputPluginInit) (void *)
+		load_external_function("pgoutput",
```

Comment:
not for this PR, but would be cool to make the decoder configurable at some point

Reply:
OK

Reply:
maybe make it a compile-time constant for now (ideally one that can be overridden via -D option)